Source/RapyutaSimulationPlugins/Public/Robots/RobotVehicle.h

```diff
@@ -43,12 +43,20 @@ class RAPYUTASIMULATIONPLUGINS_API ARobotVehicle : public ARRRobotBaseVehicle
     ARobotVehicle(const FObjectInitializer& ObjectInitializer);

```

Comment:
out of topic of this pr
- can we make this class name SkeltalRobotVehicle?

Reply:
We should, though should be prudent since this class has BP child thus the changes would automatically invalidate them and require rework on BP classes. A separate PR should be dedicated for it.